### PR TITLE
Advanced camera consoles use a list for z lock rather than a single z level

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Fixes https://github.com/tgstation/tgstation/issues/30618

### CODE REVIEW

File: code/modules/shuttle/navigation_computer.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /obj/machinery/computer/camera_advanced/shuttle_docker
 	name = "navigation computer"
 	desc = "Used to designate a precise transit location for a spacecraft."
-	z_lock = ZLEVEL_STATION_PRIMARY
+	z_lock = list(ZLEVEL_STATION_PRIMARY)
```
Comment (KorPhaeron) [inline code/modules/shuttle/navigation_computer.dm]: @Cyberboss 

What is the sanest way to make this use GLOB.z_levels_station

Late initialize and then update the list?

Comment (MoreRobustThanYou) [reply code/modules/shuttle/navigation_computer.dm]: i think there's a hidden function that's run called init()

so you could do, i think

```
z_lock = GLOB.z_levels_station
```

**Comment by ExcessiveUseOfCobblestone** [comment]
oh yeah I was suppose to make borgs unable to use those.

Guess that meme got kicked to the curb since no one complains about it anymore so :man_shrugging:

File: code/modules/shuttle/navigation_computer.dm (hunk 1 above)
Comment (Cyberboss) [reply code/modules/shuttle/navigation_computer.dm]: no, just set it in Initialize()

File: code/game/gamemodes/miniantags/abduction/machinery/camera.dm [hunk 2]
```diff
@@ -9,7 +9,7 @@
 	var/datum/action/innate/vest_disguise_swap/vest_disguise_action = new
 	var/datum/action/innate/set_droppoint/set_droppoint_action = new
 	var/obj/machinery/abductor/console/console
-	z_lock = ZLEVEL_STATION_PRIMARY
+	z_lock = list(ZLEVEL_STATION_PRIMARY)
```
Comment (Cyberboss) [inline code/game/gamemodes/miniantags/abduction/machinery/camera.dm]: Make this list a copy of the GLOB list in Initialize()

File: code/modules/shuttle/navigation_computer.dm (hunk 1 above)
Comment (Cyberboss) [reply code/modules/shuttle/navigation_computer.dm]: Ditto

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
@KorPhaeron

**Comment by KorPhaeron** [comment]
:tada:

**Pull request merged**